src/lex/mod.rs

```diff
@@ -171,7 +171,7 @@ impl Lexer {
         if self.parse_int(radix)?.is_none()
             && !(radix == Radix::Octal || radix == Radix::Decimal || self.peek() == Some('.'))
         {
-            return Err(LexError::MissingDigits(radix.try_into().unwrap()));
+            return Err(LexError::MissingDigits(radix));
```

Comment:
This was fixed on master :/ if you rebase it will fix the rest of the clippy lints.

Reply:
I tried rebasing (I just squashed everything together first because I move things several times, sometimes pointlessly) and still had to manually fix this one. Would you prefer the merge commit as it is or a rebased squash commit?

Reply:
Oh I missed that you merged because the diff is so big. Yeah that's fine.